### DISCUSSION TITLE
How to update with Associations

### DISCUSSION THREAD

**Discussion by MichelFonseca1024** [sequelize/sequelize#14014] [Help & Questions] [answered]
I managed to do the insert with relationships, but I can’t update. In the [documentation](https://sequelize.org/v6/manual/creating-with-associations.html) it shows how to create with relationships, however it doesn't say how or if it is possible to update.

* sequelize v 6.6

**Comment by ephys** [comment, marked as answer]
The `Model#save`, `Model.update`, and `Model#update` methods are not aware of associations, but you can call `save` on the associated entity itself
